airflow/config_templates/default_airflow.cfg

```diff
@@ -240,7 +240,7 @@ fail_fast = False
 
 [api]
 # How to authenticate users of the API
-auth_backend = airflow.api.auth.backend.default
+auth_backend = airflow.api.auth.backend.deny_all
```

Comment:
Can you add note in UPDATING.MD? Some may install AIRFLOW automatically and may be surprised that the default value has changed.  I also think that he should change the name of package `airflow.api.auth.backend.deny_all  to a different one to better describe the content.
I also noticed that another person(me) is already working on it.
https://github.com/apache/airflow/pull/6625/files